Gemfile

```diff
@@ -28,6 +28,8 @@ group :development, :test do
   gem 'capybara'
   gem 'byebug'
   gem 'pry'
+  gem 'brakeman', require: false
```

Comment:
We are using Code Climate, but for "reasons" I don't see any security scanning available. Here's our Code Climate page: https://codeclimate.com/github/18F/micropurchase.